index.html

```diff
@@ -59,6 +59,47 @@
         </picture>
     </div>
 </div>
+
+<h2 style="text-align: center; font-size: 1.4em;">Getting more done <span style="color: #b0197e;">after 2am</span> than most people do all day.</h2>
+
+<!-- Triple image set -->
+<div class="width_capped">
+    <div class="container">
+        <div id="triple_image_wrapper">
+        <article class="card" role="article">
+            <a class="triple_a" href="/about/">
```

Comment:
Did you not want these to be _blank? If so then it's fine either way, just making sure

Reply:
I think they work pretty well as is, what would _blank change about them?